spikesorters/basesorter.py

```diff
@@ -57,6 +58,9 @@ def __init__(self, recording=None, output_folder=None, verbose=False,
         self.parallel = parallel
         self.params = self.default_params()
 
+        if self.parallel:
+            assert self.compatile_with_parallel_thread, 'this sorter do not accept parallel=True for groups'
```

Comment:
do -> does
Also this is not only for groups, but more generally for a gropuing_property

Reply:
@samuelgarcia can you change this?

Reply:
I did it already. No ?

Reply:
Yes! Sorry I missed it. We can merge then :)